src/LowerParallelTasks.cpp

```diff
@@ -275,6 +278,8 @@ struct LowerParallelTasks : public IRMutator {
                 closure_args[2] = make_scalar_arg<int32_t>(loop_extent_name);
                 closure_args[3] = closure_arg;
                 closure_args[4] = make_scalar_arg<void *>(closure_task_parent_name);
+
```

Comment:
Seems a little odd to have a blank line here but the comment line up above.

Reply:
Done